src/Auth/Login.tsx

```diff
@@ -3,16 +3,23 @@ import "./Auth.css";
 import logo from "../images/logo2X.png";
 import google from "../images/google-logo.png";
 import kakao from "../images/kakao-logo.png";
-import { Link } from "react-router-dom";
+import { Link, useLocation, useNavigate } from "react-router-dom";
 import useFormValidation from "./useFormValidation";
+import { postSignIn } from "../api";
 
 const Login = () => {
+  const navigate = useNavigate();
+  const location = useLocation();
+  const { email: initalEmail, password: initalPassword } = location.state || {};
```

Comment:
아래의 useState에서 기본값을 빈 스트링으로 주는 것보다, 가져올 때 부터 기본값을 설정하는 것이 유지보수에 유리해서 추천드려요.

```suggestion
  const { email: initalEmail = "", password: initalPassword = "" } = location.state || {};
```